src/Illuminate/Bus/DatabaseBatchRepository.php

```diff
@@ -286,9 +286,7 @@ public function pruneUnfinished(DateTimeInterface $before)
      */
     public function transaction(Closure $callback)
     {
-        return $this->connection->transaction(function () use ($callback) {
-            return $callback();
-        });
+        return $this->connection->transaction(fn () => $callback());
```

Comment:
This line can be shortened further, removing the extra stack call:

```php
return $this->connection->transaction($callback);
```

which would also allow `$callback` to optionally accept the `Illuminate\Database\Connection` instance as an argument.

https://github.com/laravel/framework/blob/76a158efd9a760da708bf957c7df1377bfce1315/src/Illuminate/Database/Concerns/ManagesTransactions.php#L28-L30

Reply:
That requires your contribution. I do not dare to go for it.